tracks/javascript/exercises/armstrong-numbers/mentoring.md

```diff
@@ -0,0 +1,47 @@
+### Reasonable solution
+
+The solution should consist of the following three steps:
+
+1. convert the input `number` to a list of digits
+2. `reduce` the list of digits to a _sum_ of each digit to the power of the number of digitis
+3. compare the input to the summation
+
+```javascript
+const digits = [...input.toString(10)].map(Number)
+const n = digits.length
+return input === digits.reduce((result, digit) => result + digit ** n, 0)
+```
```

Comment:
Can we put this in a function? It is better to have a `complete` solution IMO.

Reply:
Woops. I don't know why it's not ❌